src/scripts/scripts/zone/coilfang_resevoir/serpent_shrine/instance_serpent_shrine.cpp

```diff
@@ -37,79 +37,27 @@ ObjectGuid ConsoleGuids[5];
 ObjectGuid BridgeConsoleGuid;
 
 
-bool UnlockVashjDoor(ScriptedInstance* instance)
-{
-    return (((instance->GetData(DATA_HYDROSS_EVENT) == SPECIAL) && 
-    (instance->GetData(DATA_LURKER_EVENT) == SPECIAL)           && 
-    (instance->GetData(DATA_LEOTHERAS_EVENT) == SPECIAL)        && 
-    (instance->GetData(DATA_KARATHRESS_EVENT) == SPECIAL)       && 
-    (instance->GetData(DATA_MOROGRIM_EVENT) == SPECIAL)));
-}
-
-void UnlockGameObject(Map* map, uint64 GameObjectGUID)
-{
-    if (GameObject *go = map->GetGameObject(GameObjectGUID)) 
-    {
-        go->RemoveFlag(GAMEOBJECT_FLAGS, GO_FLAG_LOCKED);
-    }
-}
-
 bool GOUse_go_vashj_console_access_panel(Player *player, GameObject* go)
 {
     ScriptedInstance* instance = (ScriptedInstance*) go->GetInstanceData();
 
     if (!instance)
-    {
         return false;
-    }
 
-    bool returnVal = true;
-
-    uint64 id = go->GetGUID();
+    if (!go)
+        return false;
 
-    if (id == ConsoleGuids[0])
-    {
-        instance->SetData(DATA_HYDROSS_EVENT, SPECIAL);
-        player->TextEmote("activates console #1 [Hydross the Unstable].");
-    }
-    else if (id == ConsoleGuids[1])
-    {
-        instance->SetData(DATA_LURKER_EVENT, SPECIAL);
-        player->TextEmote("activates console #2 [The Lurker Below].");
-    }
-    else if (id == ConsoleGuids[2])
+    if (go->GetGUID() == BridgeConsoleGuid) 
     {
-        instance->SetData(DATA_LEOTHERAS_EVENT, SPECIAL);
-        player->TextEmote("activates console #3 [Leotheras the Blind].");
-    }
-    else if (id == ConsoleGuids[3])
-    {
-        instance->SetData(DATA_KARATHRESS_EVENT, SPECIAL);
-        player->TextEmote("activates console #4 [Fathom-Lord Karathress].");
-    }
-    else if (id == ConsoleGuids[4])
-    {
-        instance->SetData(DATA_MOROGRIM_EVENT, SPECIAL);
-        player->TextEmote("activates console #5 [Morogrim Tidewalker].");
-    }
-    else if (id == BridgeConsoleGuid) {
         instance->SetData(DATA_BRIDGE_CONSOLE, DONE);
-        player->TextEmote("activates console #6 [Access to Lady Vashj].");
+        player->TextEmote("opened the door and created the platform to Lady Vashj.");
```

Comment:
Cause the numbering system #6 is void since you dont get to see 1 through 5, hence why I changed it